Src/Plugins/Tests.Security/Validators/SEC101_017.NpmAuthorTokenValidatorTests.cs

```diff
@@ -66,15 +66,15 @@ public void NpmAuthorTokenValidator_MockHttpTests()
     ""total"": 1,
     ""urls"": {}
 }
```

Comment:
1. convert this to the object inside the class.
2. Make the class internal so u can re-use it.
3. add a new test to cover the exception part